### ISSUE TITLE
Serverless (Laravel) Microservice on AWS

### ISSUE THREAD

**Issue by benfreke** [BrisPHP/meetups#19]
We've had some success with this, but only after quite a few challenges. Some technical, some business related.

Would be keen to do a talk on this around:

Why we wanted a microservice
Code challenges
Infrastructure challenges
Monitoring challenges

And how we solved / are solving those problems.

Additionally, in 3 months time we should have some dollar values to talk about whether serverless was cheaper for us too.

**Comment by dpdme** [comment]
Hey @benfreke Are you still keen to drop some wisdom n this at the next meetup mate?

**Comment by benfreke** [comment]
@dpdme I'm not certain how much wisdom there will be, but certainly happy to present. I think there are some really good learnings that will be of interest to the community.

**Comment by benfreke** [comment]
Hi @dpdme and @ndench 

I'm going to have to pull out sorry. I'm not going to be able to pull together the information for this talk in time.

**Comment by ndench** [comment]
Update - we're going to schedule the next meetup on the 29th of January.

**Comment by benfreke** [comment]
Hi @ndench 
I'm just checking with my co-presenters and I'll get back to you asap

**Comment by benfreke** [comment]
@ndench 
I'm not going to be able to present, I had a look at my notes over the break and I simply don't have enough information to put together a presentation. And neither did my co-presenter. 

Once I have something useful to share with the community I'll create a new issue.
